consul_template/templates/consul-template.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Consul Template
+Requires=docker.service
+After=docker.service
+
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker stop consul-template
```

Comment:
I admit I had some troubles parsing the double negations in https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStartPre= - but if I understand correctly `docker run` should be called even is stopping and removing fails?